src/webui/www/public/scripts/login.js

```diff
@@ -52,7 +52,7 @@ const submitLoginForm = (event) => {
         .then(async (response) => {
                 const responseText = await response.text();
                 if (response.ok && (responseText === "Ok."))
-                    location.replace(location); // redirect
+                    location.reload(); // reload after login
```

Comment:
Will this work for you? Please test it:
```js
if (response.ok && (responseText === "Ok.")) {
  location.replace(location); // redirect
  location.reload(true);
}
```

Reply:
> Will this work for you? Please test it:
> 
> ```js
> if (response.ok && (responseText === "Ok.")) {
>   location.replace(location); // redirect
>   location.reload(true);
> }
> ```

yes. this code works. but:

```
location.reload(true); //'true' is unnecessary.

location.reload(); //is correct.
```

see documentation:
https://developer.mozilla.org/en-US/docs/Web/API/Location/reload

Reply:
>yes. this code works

Do you mind update this PR to that code?

Reply:
> > yes. this code works
> 
> Do you mind update this PR to that code?

PR updated. Thanks